gateway-service-impl/src/test/java/org/hypertrace/gateway/service/explore/RequestHandlerTest.java

```diff
@@ -54,8 +54,8 @@ public void orderByExpressionsWithFunction_shouldMatchCorrespondingSelections()
                                         LiteralConstant.newBuilder()
                                             .setValue(
                                                 Value.newBuilder()
-                                                    .setValueType(ValueType.LONG)
-                                                    .setLong(30))))))
+                                                    .setValueType(ValueType.STRING)
```

Comment:
I think none of the existing tests will change. But, we will have to add new tests where requests can take iso string as well.

Reply:
should have read the other comments first - I repeated this in a few other spots 😉

Reply:
Now, you have added a test for iso duration request explicitly, can we revert this back to the existing argument (Long one)?